Source/RageCommunity.Library/Wrappers/NativeAudioWrappers.cs

```diff
@@ -6,6 +6,12 @@ namespace RageCommunity.Library.Wrappers
 {
     public static partial class NativeWrappers
     {
-        
+        /// <summary>
+        /// Varies the horn sound for a vehicle.
+        /// </summary>
+        public static void SetVehicleHornVariation(Vehicle vehicle, int variation)
```

Comment:
Do you have any info on the variation values?  I tried searching, but didn't find anything.

Reply:
No, sorry. If we want to add a detailed description we must do some research by our self.